Description: docbook2x-man does not create output that lintian is delighted by
Author: Thorsten Alteholz <debian@alteholz.de>

Index: ent-1.2debian/Makefile
===================================================================
--- ent-1.2debian.orig/Makefile	2018-12-12 13:49:49.300294267 +0100
+++ ent-1.2debian/Makefile	2018-12-12 13:50:05.736293774 +0100
@@ -2,11 +2,12 @@
 #   Unix make file for random test program
 
 CC = gcc
-CFLAGS = -g -Wall
+CFLAGS += -g -Wall
 #CFLAGS = -O
+LDFLAGS += -Wl,-z,now
 
-ent:	ent.o iso8859.o randtest.o chisq.o
-	$(CC) $(CFLAGS) ent.o iso8859.o randtest.o chisq.o -o ent -lm
+ent:	ent.o iso8859.o randtest.o chisq.o ent.1
+	$(CC) $(CFLAGS) $(LDFLAGS) $(CPPFLAGS) ent.o iso8859.o randtest.o chisq.o -o ent -lm
 
 ent.c:	iso8859.h randtest.h
 
@@ -31,3 +32,16 @@
 
 clean:
 	rm -f *.o *.bak ent ent.zip
+
+%.1: %.1.txt
+	asciidoc -b docbook -d manpage $<
+	docbook2x-man $*.1.xml
+	mv $*.1 $*.1.pre
+	cat $*.1.pre | sed "s/-b /\\\\-b/" | \
+		sed "s/-c /\\\\-c/" | \
+		sed "s/-f /\\\\-f/" | \
+		sed "s/-t /\\\\-t/" | \
+		sed "s/-u /\\\\-u/" | \
+		sed "s/ -b/ \\\\-b/" > $*.1
+	rm -f $*.1.xml $*.1.pre
+
